src/Core/JVM/PHPClassInvoker.php

```diff
@@ -0,0 +1,132 @@
+<?php
+namespace PHPJava\Core\JVM;
+
+use PHPJava\Core\JavaClassInterface;
+use PHPJava\Core\JavaSimpleClass;
+use PHPJava\Core\JVM\Field\PHPDynamicField;
+use PHPJava\Core\JVM\Field\PHPStaticField;
+use PHPJava\Core\JVM\Invoker\PHPClassDynamicMethodInvoker;
+use PHPJava\Core\JVM\Invoker\PHPClassStaticMethodInvoker;
+use PHPJava\Packages\java\lang\_Object;
+
+class PHPClassInvoker implements ClassInvokerInterface
+{
+    use Extended\ProviderProvidable;
+    use Extended\JavaClassProvidable;
+    use Extended\DynamicAccessorProvidable;
+    use Extended\StaticAccessorProvidable;
+
+    /**
+     * @var array
+     */
+    private $dynamicMethods = [];
```

Comment:
Probably `_MethodInfo[][]`?

Reply:
@chitoku-k 
No, It isn't a `_MethodInfo` because PHPClass haven't a Constant Pool.

Reply:
Ah, I had a mistake when reading this part. How about `\ReflectionMethod[][]` then?

Reply:
Sorry 🙇‍♀  It is auto-generated by my IDE (IntelliJ IDEA).
Yes, you are correct. I'll change it 👍